policies/SI-System_and_Information_Integrity/component.yaml

```diff
@@ -44,3 +43,46 @@
         need to address the personnel or roles within the organization who
         require notificiation.
 
+- control_key: SI-8
+  standard_key: NIST-800-53
+  covered_by: []
+  implementation_status: none
+  narrative:
+    - key: a
+      text: |-
+        The organization will be responsible for employing spam protection
+        mechanisms at organizational information system entry and exit points
+        to detect and take action on unsolicited messages. A successful
+        control response will need to address how the organization deploys
+        spam protection.
+
+    - key: b
+      text: |-
+        The organization will be responsible for updating spam protection
+        mechanisms when new releases are available in accordance with
+        organizational configuration management policy and procedures. A
+        successful control response will describe the organizations process
+        to ensure spam protection mechanisms are updated.
+
+- control_key: SI-8 (1)
+  standard_key: NIST-800-53
+  covered_by: []
+  implementation_status: none
+  narrative:
+    - text: |-
+        The organization is responsible for centrally managing spam protection
+        mechanisms. A successful control response will indicate how spam
+        protection is provided as a corporate service, and when applicable,
+        how spam protection content is disseminated to subordinate mail
+        servers.
+
+- control_key: SI-8 (2)
+  standard_key: NIST-800-53
+  covered_by: []
+  implementation_status: none
+  narrative:
+    - text: |
```

Comment:
missing `-`, probably because it somehow ended up getting added to the `orga-nization` line lol

Reply:
Thanks! fixed.